numpy/lib/tests/test_function_base.py

```diff
@@ -1498,6 +1500,56 @@ def test_size_zero_output(self):
             f(x)
 
 
+class TestLeaks(object):
+    class A(object):
+        iters = 20
+
+        def bound(self, *args):
+            return np.int(0)
```

Comment:
`np.int` is just `int`. Are you sure you want this?

Reply:
changing to `return 0`